components/StoryCard/StoryCard.styles.tsx

```diff
@@ -1,15 +1,35 @@
 import { cnb } from 'cnbuilder';
 
-export const root = '@container relative z-10 max-w-[32rem] sm:max-w-400 md:max-w-full mx-auto';
-
-export const cardWrapper = 'relative group @200:text-15 @250:text-17 @280:!type-0 @md:!text-26';
+export const root = (isHorizontal: boolean) => cnb(
+  '@container relative z-10 mx-auto', {
+  'max-w-300 sm:max-w-400 md:max-w-full': !isHorizontal,
+  'max-w-700 lg:max-w-none': isHorizontal,
+  },
+);
```

Comment:
@yvonnetangsu I think a demo of this convention would be worth sharing in an upcoming dev chat as more of our team is getting into tw and react.

Reply:
Sure I can do that @sherakama 😄